mailpoet/tests/integration/Segments/WooCommerceTest.php

```diff
@@ -660,7 +660,7 @@ private function cleanData(): void {
       DELETE FROM
         {$subscribersTable}
       WHERE
-        email LIKE 'user-sync-test%'
+        email LIKE 'user-sync-test%' OR email = ''
```

Comment:
Do we need to use WooCommerce functions instead of the direct queries below that delete orders and order meta?

https://github.com/mailpoet/mailpoet/pull/4343/commits/f8062f1ccd554d1494793568127bd0f2ddd47e15#diff-cd3b372ece0ad4e6702c5742311661aaff68b14de05e9fb8eb3fa5a5c4d50368L665-L683

I wonder if we need to do the same for \MailPoet\Test\DataFactories\WooCommerceOrder::create()?

https://github.com/mailpoet/mailpoet/blob/9f790efbf0f032fc32e264dc33450286b362976e/mailpoet/tests/DataFactories/WooCommerceOrder.php#L102